setup.py

```diff
@@ -0,0 +1,4 @@
+# This file exists to keep dependabot happy:
+# https://github.com/dependabot/dependabot-core/issues/4483
+from setuptools import setup
+setup()
```

Comment:
Out of curiosity, does `setup()` read from setup.cfg?

Reply:
It should, to my knowledge... but I want to still confirm this

Reply:
as far as I can tell this works fine: the wheels dist-info contains the correct METADATA.

(as a sidenote, we may be missing a license variable as metadata contains `License: UNKNOWN`)